test/global-table.test.ts

```diff
@@ -0,0 +1,53 @@
+import { StackProps, aws_dynamodb } from 'aws-cdk-lib';
+import * as cdk from 'aws-cdk-lib';
+import { Template } from 'aws-cdk-lib/assertions';
+import * as ddb from 'aws-cdk-lib/aws-dynamodb';
+import { GlobalTable } from '../src/global-table';
+
+test('global table region is matched to stack region, when unspecified', () => {
```

Comment:
This name describes just one property that needs to be verified in this test. Rename to something more broad for testing the "happy path" or "default scenario". 
```suggestion
test('Create global table with partition key and default properties', () => {
```